docs/static/docker.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[docker]]
 === Running Logstash on Docker
 Docker images for Logstash are available from the Elastic Docker
-registry. The base image is https://hub.docker.com/_/centos/[centos:7].
+registry. The base image is https://hub.docker.com/_/ubuntu[ubuntu:20.04].
```

Comment:
```suggestion
registry. The base image is https://hub.docker.com/_/ubuntu[ubuntu], currently `ubuntu:20.04`.
```

Reply:
Thanks for this change, @Tibo669. I've suggested a minor tweak to make the link text more accurately reflect the link destination.  This tweak would be more user friendly for people using screen readers.